### ISSUE TITLE
Can I set the dropout layers on/off for a trained model used in another model as a layer?

### ISSUE THREAD

**Issue by saintthor** [keras-team/keras#12778]
I trained model0 that contains some dropout layers.
I add model0 into model1 as a layer.
When training model1, will the dropout layers in model0 work? How to turn on or turn off them?

**Comment by ymodak** [comment]
Hi!
This issue isn't related to a bug/enhancement/feature request or other accepted types of issue.
To ask questions, please see the following resources :
[StackOverflow](http://stackoverflow.com/questions/tagged/keras)
[Keras Slack channel](https://keras-slack-autojoin.herokuapp.com)
[Keras.io support](https://keras.io/#support)
[Gitter](https://gitter.im/Keras-io/Lobby)
Thanks!
If you think we made a mistake, please open another issue explaining your request in detail.
